src/Matmar10/Money/Entity/Money.php

```diff
@@ -79,17 +112,17 @@ public function getCurrency()
 
     public function getScale()
     {
-        return $this->scale;
+        return $this->scale ?: $this->scale = bcpow(10, $this->currency->getPrecision(), 0);
```

Comment:
Ah OK, that makes sense. Can you change it to be less clever?

```
    if (!$this->scale) {
        $this->scale = bcpow(10, $this->currency->getPrecision(), 0);
    }

    return $this->scale;
```

The line as it's written now is harder to groc.